README.md

```diff
@@ -623,6 +626,29 @@ Url::make('Link', 'url')
     ->conditionalLogic([
         ConditionalLogic::where('type', '==', 'link')
     ]),
+
+// "and" condition
+Textarea::make('Embed code', 'embed')
+    ->conditionalLogic([
+        ConditionalLogic::where('type', '!=', 'document')->and('type', '!=', 'link')
+    ]),
+// use multiple ConditionalLogic for "or" condition
+Text::make('Title', 'title')
+    ->conditionalLogic([
+        ConditionalLogic::where('type', '!=', 'document')
+        ConditionalLogic::where('type', '!=', 'link')
+    ]),
+```
+
+It is possible to interact with a field group other than the one of the conditional field by specifying the group by passing an associative array in this way:
+
+```php
+use Extended\ACF\Fields\Text;
+
+Text::make('Highlight title', 'highlight_title')
+    ->conditionalLogic([
+        ConditionalLogic::where(['group' => 'other-group', 'name' => 'enable_highlight'], '==', 'on')
```

Comment:
Is the `group` equal to the field group name?

Reply:
Yes this is the group name :)